app/bridge/client.js

```diff
@@ -47,19 +47,6 @@ function createNewClient(name, version) {
 
 
   /*
-   * Deferred
-   */
-  function Deferred() {
-    var deferred = {};
-    deferred.promise = new Promise(function(resolve, reject) {
-      deferred.resolve = resolve;
-      deferred.reject = reject;
-    });
-    return deferred;
-  }
-
-
-  /*
```

Comment:
I understand why you did this. And while it is cool for other files of the app, I think we don't want this library to rely on a library that belong to the application (promises.js).
So let's keep this code here for now, but it is fine if you use Promise.defer into other places.

Fwiw, one of the next step will likely be to move to thread.js instead of keeping around this old version of the bridge, so it is not that important :)